cdk/README.md

```diff
@@ -0,0 +1,15 @@
+# Welcome to your CDK TypeScript project!
+
```

Comment:
Might be worth adding some detail about how the stacks are set up. V. happy to contribute this change on Mon, or pair.